dags/iop/parser.py

```diff
@@ -100,3 +105,80 @@ def _get_extracted_arxiv_eprint_value(self, article):
             self.logger.error("The arXiv value is not valid.", dois=self.dois)
         except AttributeError:
             self.logger.error("No arXiv eprints found", dois=self.dois)
+
+    def _get_authors(self, article):
+        contrib_types = article.findall(
+            "front/article-meta/contrib-group/contrib[@contrib-type='author']"
+        )
+        authors = []
+        surname = ""
+        given_names = ""
+
+        for contrib_type in contrib_types:
+            try:
+                surname = contrib_type.find("name/surname").text
+            except AttributeError:
+                self.logger.error("Surname is not found in XML", dois=self.dois)
+            try:
+                given_names = contrib_type.find("name/given-names").text
+            except AttributeError:
+                self.logger.error("Given_names is not found in XML", dois=self.dois)
+
+            reffered_ids = contrib_type.findall("xref[@ref-type='aff']")
+            affiliations = [
+                self._get_affiliation_value(article, reffered_id)
+                for reffered_id in reffered_ids
+                if self._get_affiliation_value(article, reffered_id) is not None
+            ]
+
+            if "collaboration" not in given_names.lower():
```

Comment:
how it is possible that `collaborations` will be in the author `given_names` entry? 😨

Reply:
yes, it is possible. IOP is putting collaborations in the author fields, where is given_name

Reply:
no, it just has a 'collaborations' in given name field